ml-agents/tests/yamato/standalone_build_tests.py

```diff
@@ -21,6 +21,7 @@ def main(scene_path, build_target):
         output_path=executable_name,
         scene_path=scene_path,
         build_target=build_target,
+        log_output_path=None,  # Log to stdout so we get timestamps on the logs
```

Comment:
I'm tempted to leave this the way it was before, and only enable this if we want to profile the build more. @surfnerd any thoughts?

Reply:
seems like a good move to me.  Not having the timestamps is a bit of a pain.